### PR TITLE
Adding in more recent mysqldump version

### PR DESCRIPTION
**Pull request by hallaroo**
This is still being validated so I will hold off on merging until it is (and is reviewed).

### CODE REVIEW

**Comment by rohit-nayak-ps** [comment]
@hallaroo, may I suggest you convert this and #594  to `draft`s since they are not yet ready. Otherwise they show up as Ready and Pending Review and it may get accidentally reviewed and merged. Thanks!

File: content/en/docs/user-guides/configuration-basic/exporting-data.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ For example to export the `commerce` keyspace using the `mysqldump` binary from
 $ mysqldump  --lock-tables=off --set-gtid-purged=OFF --no-tablespaces commerce > commerce.sql
 ```
 
+To do the same, but from Percona 8.0:
```
Comment (mattlord) [inline content/en/docs/user-guides/configuration-basic/exporting-data.md:39]: We should change this to MySQL 8.0 (Percona 8.0 is identical in this context).

File: content/en/docs/user-guides/configuration-basic/exporting-data.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ For example to export the `commerce` keyspace using the `mysqldump` binary from
...
+
+```sh
+$ mysqldump  --lock-tables=off --add-locks=false --set-gtid-purged=OFF --no-tablespaces --add-drop-table=false -h x.x.x.x -u db_username -p db_name
```
Comment (mattlord) [inline content/en/docs/user-guides/configuration-basic/exporting-data.md:42]: From the paragraph above, I'm guessing that this should be:
```
$ mysqldump  --skip-network-timeout --skip-lock-tables --skip-add-locks --set-gtid-purged=OFF --no-tablespaces commerce > commerce.sql 
```

But I don't know the context for this... @aquarapid do you remember?

**Comment by hallaroo** [comment]
@mattlord verified what works for both 5.7 and 8.0. Which is the current example with --add-drop-table=false added. I will open a new PR to make that change.

**Pull request closed without merging**